k256/src/arithmetic/field/field_10x26.rs

```diff
@@ -0,0 +1,681 @@
+use elliptic_curve::subtle::{Choice, ConditionallySelectable, ConstantTimeEq, CtOption};
+
+#[cfg(feature = "getrandom")]
+use getrandom::getrandom;
```

Comment:
There isn't currently a `getrandom` feature... see #47 (and #62 where I got rid of a few usages that were missed)

Using `CryptoRng + RngCore` is a bit friendlier to embedded devices as it works with non-global RNGs. `rand_core` also has a `getrandom` feature you can use to access the OS RNG via `rand_core::OsRng`

Reply:
Fixed